zerver/lib/bugdown/__init__.py

```diff
@@ -146,6 +146,7 @@ def add_a(root, url, link, title=None, desc=None,
         a.set("data-id", data_id)
     img = markdown.util.etree.SubElement(a, "img")
     img.set("src", url)
+    img.set("alt", title)
```

Comment:
and i'd imagine that the hover-title behaviour would only be able to provide users with the title of the image, which is not as useful for images as it is for emoji codes